src/openstudio_app/OpenStudioApp.cpp

```diff
@@ -1288,7 +1288,7 @@ void OpenStudioApp::startMeasureManagerProcess() {
   tcpServer.close();
 
   QString portString = QString::number(port);
-  QString urlString = "http://127.0.0.1:" + portString;
+  QString urlString = "http://localhost:" + portString;
```

Comment:
@jmarrec what do you think about this?  For some reason when I run the app from the installer it can't find the measure manager.  Is there some reason the new measure manager is only listening to localhost and not 127.0.0.1?  I always thought the two were the same for IPv4.

```
macumber@Automatic-Magic-PC MINGW64 /c/openstudioapplication-1.7.0-rc0/bin
$ ./openstudio classic measure -s 1616
[2023-12-18 20:53:58] INFO  WEBrick 1.6.0
[2023-12-18 20:53:58] INFO  ruby 2.7.2 (2020-10-01) [x64-mswin64_140]
[2023-12-18 20:53:58] INFO  WEBrick::HTTPServer#start: pid=17592 port=1616
127.0.0.1 - - [18/Dec/2023:20:54:15 Mountain Standard Time] "GET / HTTP/1.1" 200 79
- -> /
[2023-12-18 20:54:35] INFO  going to shutdown ...
[2023-12-18 20:54:35] INFO  WEBrick::HTTPServer#start done.
┌────────────────────────────────────────────────────────────────────────────────┐
│  The `classic` command is deprecated and will be removed in a future release   │
└────────────────────────────────────────────────────────────────────────────────┘

macumber@Automatic-Magic-PC MINGW64 /c/openstudioapplication-1.7.0-rc0/bin
$ ./openstudio measure -s 1616
MeasureManager ReadyAccepting requests on: http://localhost:1616/

```

Reply:
You're on windows I assume? I kinda just gave up trying to make both localhost and 127.0.0.0 work on windows... It's a limitation of the cpprestsdk afaik. https://github.com/NREL/OpenStudio/blob/05c60768c1e1057083ce644d49711f7377f8ce1e/src/cli/MeasureManager.cpp#L563-L569